webapi/src/test/scala/org/knora/webapi/models/FileModels.scala

```diff
@@ -0,0 +1,1131 @@
+/*
+ * Copyright © 2021 Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.knora.webapi.models
+
+import org.knora.webapi.{ApiV2Complex, IRI}
+import org.knora.webapi.messages.{OntologyConstants, SmartIri, StringFormatter}
+import org.knora.webapi.messages.v2.responder.resourcemessages.{CreateResourceV2, CreateValueInNewResourceV2}
+import org.knora.webapi.messages.v2.responder.valuemessages.{
+  BundleFileValueContentV2,
+  DocumentFileValueContentV2,
+  FileValueV2,
+  StillImageFileValueContentV2,
+  ValueContentV2
+}
+import org.knora.webapi.sharedtestdata.SharedTestDataADM
+import org.knora.webapi.messages.IriConversions._
+import org.knora.webapi.messages.admin.responder.projectsmessages.ProjectADM
+
+import java.time.Instant
+import java.util.UUID
+
+/**
+ * Constants for use in FileModels.
+ */
+object FileModelConstants {
+
+  /** "DocumentRepresentation" */
+  val documentRepresentation = "DocumentRepresentation"
+
+  /** "TextRepresentation" */
+  val textRepresentation = "TextRepresentation"
+
+  /** "StillImageRepresentation" */
+  val stillImageRepresentation = "StillImageRepresentation"
+
+  /** "MovingImageRepresentation" */
+  val movingImageRepresentation = "MovingImageRepresentation"
+
+  /** "AudioRepresentation" */
+  val audioRepresentation = "AudioRepresentation"
+
+  /** "BundleRepresentation" */
+  val bundleRepresentation = "BundleRepresentation"
+
+  /** "knora-api" */
+  val knoraApiPrefix = "knora-api"
+
+  /** "0001" */
+  val anythingShortcode = "0001"
```

Comment:
IMO describing very descriptive field with `/** "Fieldname" */` is redundant.

Reply:
removed